### DISCUSSION TITLE
Remote Access?

### DISCUSSION THREAD

**Discussion by oaklandm** [FreshRSS/FreshRSS#3503] [General]
Can anyone point me in the direction of how to setup remote access? I have it running fine when connected to home wifi but that's it. I have it running on Ubuntu Server 20.10 on a raspberry pi4 and used nginx.

Thanks In Advance

**Comment by Frenzie** [comment]
That'd be a matter of port forwarding in the NAT/firewall settings in your router. Be very careful exposing things to the internet though.

**Reply by oaklandm** [reply]
Do you know what ports need forwarding? I forwarded fort 3306 for MySQL and thought that would be it but still doesn't allow remote access

**Reply by Alkarex** [reply]
Only port 443 (HTTPS) should be forwarded - or port 80 (HTTP) but in this case you have no transport security.
In any case, your database is not supposed to be exposed on the public Internet.
For HTTPS, you can consider using a proxy with automatic HTTPS such as Traefik.
https://github.com/FreshRSS/FreshRSS/issues/2918#issuecomment-616668682
(I will soon update our official documentation: https://github.com/FreshRSS/FreshRSS/issues/3351 )
Otherwise, you can also have a look at the YunoHost project https://yunohost.org/ to help you with a do-it-yourself hosting.
